go-controller/pkg/cluster/node.go

```diff
@@ -123,7 +123,12 @@ func (cluster *OvnClusterController) StartClusterNode(name string) error {
 		}
 	}
 
-	err = setupOVNNode(name)
+	if node, err = cluster.Kube.GetNode(name); err != nil {
+		logrus.Errorf("error retrieving node %s: %v", name, err)
+		return err
+	}
```

Comment:
@juanluisvaladas can we just do `return fmt.Errorf("error retrieving kubernetes node object %s: %v", name, err)